package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ardrive-cli",
-	"version": "1.0.2",
+	"version": "1.0.3",
```

Comment:
Nice! Though we should have done this immediately after the last release.

Reply:
Should we do a version PR to dev after each release?